test/WebSites/FormatterWebSite/Controllers/TryValidateModelController.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace FormatterWebSite
+{
+    public class TryValidateModelController : Controller
+    {
+        [HttpGet]
+        public IActionResult GetInvalidUser()
+        {
+            var user = new User()
+            {
+                Id = 0,
+                Name = "x"
+            };
+            if(!TryValidateModel(user))
+            {
+                return Content(ModelState["Id"].Errors[0].ErrorMessage + "," +
```

Comment:
This might not surface any additional error messages that could be added to the `ModelState`. Could you use `BadRequestObjectResult` here and deserialize as json on the client?